### PR TITLE
Adds tiny shards

### PR DESCRIPTION
**Pull request by nicbn**
[Changelogs]: #
:cl:
add: Destroying windows will now spawn tiny shards
/:cl:

### CODE REVIEW

**Comment by Kromgar** [comment]
Nanotrasen Tempered Safety Glass when?

**Comment by GunHog** [comment]
It should hurt to step on it barefoot.

**Comment by nicbn** [comment]
This is for decoration only, shouldn't change anything in gameplay

**Comment by Bawhoppen** [comment]
Key janitor balance change

**Review by vuonojenmustaturska — DISMISSED** [review]
Please show us a picture of this in-game

**Comment by nicbn** [comment]
image

File: code/game/objects/effects/decals/cleanable/misc.dm [hunk 1]
```diff
@@ -25,6 +25,11 @@
 	. = ..()
 	reagents.add_reagent("ash", 30) //double the amount of ash.
 
+/obj/effect/decal/cleanable/glass
+	name = "tiny shards"
+	desc = "Back to sand."
+	icon = 'icons/obj/shards.dmi'
+	icon_state = "tiny"
```
Comment (coiax) [inline code/game/objects/effects/decals/cleanable/misc.dm]: So they're tiny shards that you can mop up, or remove with space cleaner?

**Review by ShizCalev — CHANGES_REQUESTED** [review]
make dir states for the sprite and just rotate it between them, then do dir = pick(GLOB.cardinals) on init to make it appear more random.

File: code/game/objects/effects/decals/cleanable/misc.dm (hunk 1 above)
Comment (nicbn) [reply code/game/objects/effects/decals/cleanable/misc.dm]: Correct

**Comment by nicbn** [comment]
Done

File: code/game/objects/effects/decals/cleanable/misc.dm [hunk 2]
```diff
@@ -25,6 +25,15 @@
...
 
+/obj/effect/decal/cleanable/glass
+	name = "tiny shards"
+	desc = "Back to sand."
+	icon = 'icons/obj/shards.dmi'
+	icon_state = "tiny"
+
+/obj/effect/decal/cleanable/glass/Initialize()
+	. = ..()
+	dir = pick(GLOB.cardinals)
```
Comment (AnturK) [inline code/game/objects/effects/decals/cleanable/misc.dm]: setDir()

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/decals/cleanable/misc.dm [hunk 3]
```diff
@@ -25,6 +25,15 @@
 	. = ..()
 	reagents.add_reagent("ash", 30) //double the amount of ash.
 
+/obj/effect/decal/cleanable/glass
```
Comment (AnturK) [inline code/game/objects/effects/decals/cleanable/misc.dm]: Also please make these qdel themselves on ex_act so they won't have much performance impact on large explosions

**Pull request merged**